en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -21,16 +26,123 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
+    2.2 Setting up the created operation
+
+    Then we will setup the `create` sequence configurations. In this operation we are going to create a `sObjects` in the Salesforce account. An `SObject` represents a specific table in the database that you can discretely query. It describes the individual metadata for the specified object. Please find the `create` operation parameters listed here.
+       
+    - **sObjectName** : Name of the sObject which we need to create in the salesforce.
+    - **fieldAndValue** : The field and value need to store in the created salesforce sObject.
+        
+    While invoking the api the above two parameters values are coming as a user input.
+    
+    Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `create` operation into the Design pane.
+    
+     <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-create.png" title="Drag and drop create operation" width="800" alt="Drag and drop create operations"/>
+    
+    To get the input values in to the API we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/) .
+
+    Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
+    
+    <img src="../../../../assets/img/connectors/salesforce-api-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
+
+    The parameters available for configuring the Property mediator are as follows:
+    
+    > **Note**: That the properties should be add to the pallet before create the operation.
+    
+    Add the property mediator to capture the `sObjectName` value.The sObjectName type can be used to retrieve the metadata for the Account object using the GET method, or create a new Account object using the POST method. In this example we are going to create a new Account object using the POST method.
+   
+    - **name** : sObjectName
+    - **expression** : json-eval($.sObject)
+    - **type** : STRING
+   
+    <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values1.png" title="Add values to capture sObjectName value" width="800" alt="Add values to capture sObjectName value"/>
+    
+    Add the property mediator to capture the `fieldAndValue` values. The fieldAndValue contains object fields and values that user need to store.
+   
+    - **name** : fieldAndValue
+    - **expression** : json-eval($.fieldAndValue)
+    - **type** : STRING
+     
+    <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values2.png" title="Add values to capture fieldAndValue value" width="800" alt="Add values to capture fieldAndValue value"/>  
+    
+3. Creating the sequence to retrive the salesforce objects created
+
```

Comment:
We need to tell here that we are defining a new sequence called `x`

Reply:
Updated.